image/image.go

```diff
@@ -370,8 +375,8 @@ func bootstrapToRootDir(tsto *ToolingStore, model *asserts.Model, opts *Options,
 			locals = append(locals, name)
 		}
 
-		// kernel/os are required for booting
-		if typ == snap.TypeKernel || typ == snap.TypeOS {
+		// kernel/os/model.base are required for booting
+		if typ == snap.TypeKernel || typ == snap.TypeOS || snapName == model.Base() {
```

Comment:
What if model.Base returns "" ?

Reply:
That should be ok, as long as snapName is not empty. I will add code to ensure this is never the case.

Reply:
you could capture baseName set to defaultCore or model.Base() != "" earlier instead